docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: '3.7'
+
+services:
+  trac:
+    image: ghcr.io/django/code.djangoproject.com:latest
+    # Comment out to use the ghcr.io image:
+    build:
+      context: ./
+      dockerfile: Dockerfile
+    command: ["/venv/bin/tracd", "--port", "9000", "-s", "trac-env"]
+    environment:
+     - TRAC_INI_database=postgres://code.djangoproject:code.djangoproject@db/code.djangoproject
+    volumes:
+      - ./trac-env/:/code/trac-env/
```

Comment:
We might wanna provide a way to map the source in for easier development

Reply:
Excellent point, I'll make this change.